src/regexp-tree.js

```diff
@@ -123,8 +123,8 @@ const regexpTree = {
    *
    * @return TransformResult object
    */
-  optimize(regexp, whitelist) {
-    return optimizer.optimize(regexp, whitelist);
+  optimize(regexp, whitelist, {blacklist} = {}) {
```

Comment:
OK, so it's not a breaking change, and existing code which pass just the whitelist will continue to work 👍 Later we can add a deprecation notice, and eventually migrate to the single param with `{whitelist, blacklist}`.